build.gradle.kts

```diff
@@ -86,6 +86,7 @@ dependencies {
     implementation("io.ktor:ktor-server-resources")
     implementation("io.ktor:ktor-server-default-headers")
     implementation("io.ktor:ktor-server-html-builder")
+    implementation("io.ktor:ktor-server-content-negotiation")
```

Comment:
we already declare this

Reply:
we had the client one, but we needed the server one too when i needed to deserialize a request body and not just query params. spent quite some time trying to figure out what wasn't working bc i thought the same that we already had what we needed